sunpy/map/tests/test_mapbase.py

```diff
@@ -1500,7 +1499,6 @@ def test_contour_units(simple_map):
         assert np.all(c1 == c2)
 
 
-@pytest.mark.skipif(Version(matplotlib.__version__) < Version("3.6.0"), reason="Fails on old MPL versions, the first with block raises a different error")
```

Comment:
Why is this test which was being skipped before is failing now? Is it a problem in the test itself or the code is incorrect somewhere?

Reply:
It was only skipped on oldest deps, it always ran on the CI. 

The test output will have changed and we need to work out why and adjust the test.